server/routerlicious/packages/services-shared/src/livenessMonitor/oldestOpChecker.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
I assume each service has its own monitor. Do we want to centralize all checkers? Why not put the checkers close to their monitors?